scripts/install/ceres_install.bash

```diff
@@ -2,6 +2,7 @@
 set -e  # exit on first error
 UBUNTU_VERSION=`lsb_release --release | cut -f2`
 SRC_PREFIX_PATH="/usr/local/src/"
```

Comment:
@yuzhangbit is there a reason why `/usr/local/src` isn't a good idea, I was merely following the [FHS standard](https://unix.stackexchange.com/questions/10814/what-is-the-usr-local-src-folder-meant-for)? I've also thought about creating our own `/opt/wavelab` I'm open to options / ideas